geospaas/base_viewer/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.gis.db import models as geomodels
+
+class SearchModelForLeaflet(geomodels.Model):
```

Comment:
Why do you create this extra model? Is it necessary to save searches in the database?

Reply:
leaflet needs this model to work with its widget. This information comes thoroughly from the user and it is not related to our model and database. Thus, it is good to use its specified model as described in the tutorial of leaflet.

Reply:
Maybe I am mistaken but I don't think it is really necessary, see my last comment on the forms.